assets/prototype/domain/shared/predicates/tickets/filters/validInfiniteQuantity.ts

```diff
@@ -1,9 +1,20 @@
+/**
+ * External dependencies
+ */
+import { is } from 'ramda';
+
+/**
+ * Internal dependencies
+ */
+import { Ticket } from '../../../../eventEditor/data/types';
+
 /**
  * @param {Object} ticket    event ticket object
  * @return {boolean} true if qty property is valid and unlimited
  */
-const validInfiniteQuantity = (ticket) => {
-	return ticket.qty === 'INF' || ticket.qty === Infinity;
+const validInfiniteQuantity = (ticket: Ticket) => {
+	const isNumber = is(Number, ticket.quantity); // This check has been added because qty is optional in Ticket type.
+	return isNumber && ticket.quantity === Infinity;
```

Comment:
needs typing for return value

Reply:
Added: https://github.com/eventespresso/event-espresso-core/pull/2150/commits/b8bebcf79946133a296c6ad4cccd52bddb17ea85